components/expenses/ExpenseFormPayeeSignUpStep.js

```diff
@@ -403,21 +365,6 @@ const ExpenseFormPayeeSignUpStep = ({ formik, payoutProfiles, collective, onCanc
             </StyledInputField>
           )}
         </FastField>
-
-        {values.payoutMethod && (
-          <Field name="payoutMethod">
-            {({ field, meta }) => (
-              <Box mt={3} flex="1">
-                <PayoutMethodForm
-                  fieldsPrefix="payoutMethod"
-                  payoutMethod={field.value}
-                  host={collective.host}
-                  errors={meta.error}
-                />
-              </Box>
-            )}
-          </Field>
-        )}
       </Grid>
       {values.payee && (
```

Comment:
Probably not directly related to this commit, but rather a previous commit, but the code block from 369 to 421 here is exactly (almost except for one commented line I think) same as lines 355-406 from `ExpenseFormPayeeStep.js`. Maybe I was thinking we could extract to separate component? 🤔

Reply:
Yes, both components are very similar, but since they were split from the previous hierarchy, I'm not very keen to share code between these two just yet.